gravitee-apim-console-webui/src/management/api/proxy/health-check/api-proxy-health-check.component.spec.ts

```diff
@@ -302,3 +302,30 @@ describe('ApiProxyHealthCheckComponent', () => {
     fixture.detectChanges();
   }
 });
+
+describe('updateEndpointsHealthCheckConfig', () => {
+  let component: ApiProxyHealthCheckComponent;
+
+  beforeEach(() => {
+    component = new ApiProxyHealthCheckComponent({} as any, {} as any, {} as any, {} as any, {} as any);
```

Comment:
Can we avoid all those any. I think that here we should keep the same test structure and use the harnesses and not directly call the components. You could even update an existing test and just add a group without any endpoint and ensure that everything is ok